api/health.yaml

```diff
@@ -0,0 +1,37 @@
+openapi: 3.0.0
```

Comment:
health route is still a route it should be part of api.yaml

Reply:
done